homeassistant/helpers/entity_platform.py

```diff
@@ -61,6 +61,37 @@
 DATA_ENTITY_PLATFORM = "entity_platform"
 PLATFORM_NOT_READY_BASE_WAIT_TIME = 30  # seconds
 
+DEVICE_INFO_TYPES = (
```

Comment:
Potentially use a dict instead of a tuple to document the different types with the dict keys.

Reply:
I had that at first, but since I wasn't using that info, I converted it to a comment.

Reply:
Fixed.